### PR TITLE
test(online/cassandra): replace gnomock with oomplay

### PR DESCRIPTION
**Pull request by lianxmfor**

### CODE REVIEW

**Comment by wfxr** [comment]
image
Less is more !

File: internal/database/test/runtime_cassandra/cassandra.go [hunk 1]
```diff
@@ -59,8 +29,25 @@ func PrepareDB() (context.Context, *gocql.Session) {
...
+	}
+}
+
 func GetOpt(keySpace string) *types.CassandraOpt {
-	opt := CassandraDbOpt
-	opt.KeySpace = keySpace
-	return &opt
+	return &types.CassandraOpt{
+		Hosts:    []string{"127.0.0.1:9042"},
+		User:     "cassandra",
```
Comment (YikSanChan) [inline internal/database/test/runtime_cassandra/cassandra.go:49]: how about "test" to be consistent

File: internal/database/test/runtime_cassandra/cassandra.go [hunk 2]
```diff
@@ -59,8 +29,25 @@ func PrepareDB() (context.Context, *gocql.Session) {
...
+		Password: "cassandra",
```
Comment (YikSanChan) [inline internal/database/test/runtime_cassandra/cassandra.go:50]: "test"

**Review by YikSanChan — APPROVED** [review]
less is more!

File: internal/database/test/runtime_cassandra/cassandra.go (hunk 1 above)
Comment (lianxmfor) [reply internal/database/test/runtime_cassandra/cassandra.go]: fixed

File: internal/database/test/runtime_cassandra/cassandra.go (hunk 2 above)
Comment (lianxmfor) [reply internal/database/test/runtime_cassandra/cassandra.go]: fixed

**Pull request merged**